src/main/java/frc/team2412/robot/util/auto/AutoAlignment.java

```diff
@@ -0,0 +1,109 @@
+package frc.team2412.robot.util.auto;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj.smartdashboard.Field2d;
+import edu.wpi.first.wpilibj.smartdashboard.FieldObject2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.team2412.robot.Robot;
+import frc.team2412.robot.Subsystems;
+import java.util.function.Supplier;
+
+public class AutoAlignment {
+	private static Robot r = Robot.getInstance();
+	private static final Subsystems s = r.subsystems;
+
+	private static Supplier<Pose2d> currentPosition = s.drivebaseWrapper::getEstimatedPosition;
+	private static Supplier<Pose2d> goalPosition =
+			() -> AutoLogic.getSelectedAutoPath().getStartPose2d();
+
+	private static final double POSITION_TOLERANCE = 0.5;
```

Comment:
I am assuming this is half a meter?  Is that accurate enough to be useful?

Reply:
Probably not, i'll lower the tolerance